Tactical/Interface Enhanced.cpp

```diff
@@ -1621,8 +1621,8 @@ void InternalInitEDBTooltipRegion( OBJECTTYPE * gpItemDescObject, UINT32 guiCurr
 
 			/////////////////// PROJECTION FACTOR / NCTH BEST LASER RANGE
 			// with the reworked NCTH code and the laser performance factor we will display BestLaserRange instead of ProjectionFactor but we still need the mouse region
-			if (UsingNewCTHSystem() && ( GetProjectionFactor( gpItemDescObject ) > 1.0 || GetBestLaserRange( gpItemDescObject ) > 0
-				&& (gGameCTHConstants.LASER_PERFORMANCE_BONUS_HIP + gGameCTHConstants.LASER_PERFORMANCE_BONUS_IRON + gGameCTHConstants.LASER_PERFORMANCE_BONUS_SCOPE != 0) ) )
+			if (UsingNewCTHSystem() && ( GetProjectionFactor( gpItemDescObject ) > 1.0 || (GetBestLaserRange(gpItemDescObject) / CELL_X_SIZE > 0

```

Comment:
adding ` / CELL_X_SIZE` doesn't change the evaluation of the condition

Reply:
It changes condition for cases when GetBestLaserRange(gpItemDescObject) returns [0..9]

Reply:
I see. The division is evaluated to an integer. Thank you for the clarification.